CMakeLists.txt

```diff
@@ -19,6 +19,10 @@ IF(NOT EXISTS "${CMAKE_SOURCE_DIR}/cmake/base.cmake")
   MESSAGE(FATAL_ERROR "\nPlease run the following command first:\ngit submodule update --init\n")
 ENDIF()
 
+# Normally, BUILD_TESTING defaults to ON. This sets the default to OFF.
+# It needs to be included before module CTest, thus before command PROJECT
+OPTION (BUILD_TESTING "" OFF)
```

Comment:
We should comply with the common rules. We should let this variable at its default value.

Reply:
I agree. I added this so that the behavior stayed closer to the current one, but it is better to stick to the standards